packages/frontend/src/stockStrings.ts

```diff
@@ -217,8 +216,16 @@ export async function updateCoreStrings() {
     [C.DC_STR_MESSAGE_ADD_MEMBER]: tx('member_x_added'),
     [C.DC_STR_YOU_REACTED]: tx('reaction_by_you'),
     [C.DC_STR_REACTED_BY]: tx('reaction_by_other'),
-    [C.DC_STR_SECUREJOIN_WAIT]: tx('secure_join_wait_timeout'),
+    [C.DC_STR_SECUREJOIN_WAIT]: 'deprecated',
```

Comment:
This is really weird. Why not just add these strings to the "ignore list" above? There is no need to show 'deprecated' to the user instead of the default core's English string.

https://github.com/chatmail/core/blob/9ec0332483a56b97d0a5a4b00e6ae642cdc670c8/src/stock_str.rs#L415-L416